coalib/results/Diff.py

```diff
@@ -432,7 +444,7 @@ def __bool__(self):
         """
         return (self.rename is not False or
                 self.delete is True or
-                self.modified != self._file)
+                self.modified != self.original)
```

Comment:
the tests did pass before when comparing `self._raw_modified()` and `self._file`, right? Could you please add a test testing the edge-case I mentioned last time? When modifying a line suddenly with `\n` characters that hadn't some before^^